src/google/protobuf/stubs/int128.h

```diff
@@ -379,6 +357,17 @@ inline uint128& uint128::operator--() {
   return *this;
 }
 
+constexpr uint128 Uint128Max() {
+  return MakeUint128((std::numeric_limits<uint64>::max)(),
+                     (std::numeric_limits<uint64>::max)());
+}
+
+}  // namespace int128_internal
+
+using absl::uint128;
```

Comment:
I think these lines must have been added by mistake?

Reply:
I added these when testing compatibility with absl::uint128 and forgot to remove them. Sorry, fixed